data/extension/init/templates/tinygo/envoy.filters.network/default/main_test.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
I see an error in the logs when I run unit tests for the generated extension:

```shell
getenvoy extension run
```

```shell
=== RUN   TestNetworkFilter_OnNewConnection
2021/03/11 10:38:05 proxy_info_log: message: this is new connection!
--- PASS: TestNetworkFilter_OnNewConnection (0.00s)
=== RUN   TestNetworkFilter_counter
2021/03/11 10:38:05 start index out of range: 0 (start) >= 0
2021/03/11 10:38:05 proxy_critical_log: failed to load config: error status returned by host: bad argument
2021/03/11 10:38:05 proxy_info_log:
2021/03/11 10:38:05 proxy_info_log: connection complete!
--- PASS: TestNetworkFilter_counter (0.00s)
PASS
ok  	my.filters.http	(cached)
```

The following lines don't seem right:
```shell
2021/03/11 10:38:05 start index out of range: 0 (start) >= 0
2021/03/11 10:38:05 proxy_critical_log: failed to load config: error status returned by host: bad argument
```

Reply:
Thanks! This was a bug in the host emulation behavior and the mistake in the error handling. fixed in https://github.com/tetratelabs/getenvoy/pull/104/commits/436d5079305ab99f16991c70c67d761911e80ebe